docs/core/tools/extensibility.md

```diff
@@ -136,49 +143,39 @@ The sample target's project file is included below for reference. It shows how t
 
-Using the custom target depends solely on how you configure it. Since it is the usual MSBuild target, it can depend on a given target, run after another target and can also be manually invoked using the `dotnet msbuild /t:<target-name>` command. 
+Using the custom target depends solely on how you configure it. Since it's a MSBuild target, it can depend on a given target, run after another target and can also be manually invoked using the `dotnet msbuild /t:<target-name>` command. 
 
-However, if you wish to provide a better user experience to your users, you can combine per-project tools and custom targets. In this scenario, the per-project tool would essentially just accept whatever needed parameters and would translate that into the required `dotnet msbuild` invocation that would execute the target. You can see a sample of this kind of synergy on the [MVP Summit 2016 Hackathon samples](https://github.com/dotnet/MVPSummitHackathon2016) repo in the [`dotnet-packer`](https://github.com/dotnet/MVPSummitHackathon2016/tree/master/dotnet-packer) project. 
+However, if you want to provide a better user experience to your users, you can combine per-project tools and custom targets. In this scenario, the per-project tool would essentially just accept whatever needed parameters and would translate that into the required [`dotnet msbuild`](dotnet-msbuild.md) invocation that would execute the target. You can see a sample of this kind of synergy on the [MVP Summit 2016 Hackathon samples](https://github.com/dotnet/MVPSummitHackathon2016) repo in the [`dotnet-packer`](https://github.com/dotnet/MVPSummitHackathon2016/tree/master/dotnet-packer) project. 
 
 ### PATH-based extensibility
 PATH-based extensibility is usually used for development machines where you need a tool that conceptually covers more 
-than a single project. The main drawback of this extensions mechanism is that it is tied to the machine where the 
+than a single project. The main drawback of this extension mechanism is that it's tied to the machine where the 
 tool exists. If you need it on another machine, you would have to deploy it.
 
 This pattern of CLI toolset extensibility is very simple. As covered in the [.NET Core CLI overview](index.md), `dotnet` driver 
-can run any command that is named after the `dotnet-<command>` convention. The default resolution logic will first 
-probe several locations and will finally fall to the system PATH. If the requested command exists in the system PATH 
+can run any command that is named after the `dotnet-<command>` convention. The default resolution logic first 
+probes several locations and finally falls to the system PATH. If the requested command exists in the system PATH 
 and is a binary that can be invoked, `dotnet` driver will invoke it. 
 
 The binary can be pretty much anything that the operating system can execute. On Unix systems, this means anything that 
```

Comment:
The first sentence, and especially "pretty much", is problematic. Perhaps "The binary must be executable." or "The file must be executable."